zebra/router-id.c

```diff
@@ -253,6 +253,34 @@ DEFUN (no_router_id,
 	return CMD_SUCCESS;
 }
 
+DEFUN (show_router_id,
+       show_router_id_cmd,
+       "show router-id [vrf NAME]",
```

Comment:
I think json output would be useful here as well :+1:

Reply:
Time to learn how I do json output ;)

Reply:
This is absolutely non-blocking request :)